packages/odyssey-react-mui/src/EmptyState.tsx

```diff
@@ -28,22 +28,36 @@ const EmptyContainer = styled("div", {
   padding: odysseyDesignTokens.Spacing5,
   textAlign: "center",
   width: "100%",
+  alignItems: "center",
 }));
 
-export type DataTableEmptyStateProps = {
+export type EmptyStateProps = {
+  /**
+   * The main heading of the empty state
+   */
   heading: string;
+  /**
+   * A descriptive text explaining more context
+   */
   text: string;
+  /**
+   * Primary call to action
+   */
   primaryButton?: ReactNode;
+  /**
+   * Secondary call to action
+   */
   secondaryButton?: ReactNode;
```

Comment:
Shouldn't these be named `primaryCallToActionComponent` and `secondaryCallToActionComponent`? "Button" is too generic. Button for what? What does it do?

In other components like `Dialog` have them labelled as `primaryCallToActionComponent`, so we should follow that nomenclature.

Reply:
I agree. Done.